packages/server/src/rooms.ts

```diff
@@ -0,0 +1,44 @@
+import { Server as IOServer, Socket } from 'socket.io';
+import { generateRoomName } from './utils';
+
+const MAX_ROOM_SIZE = 2;
+
+export async function createRoom(io: IOServer, socket: Socket) {
+  const roomName = generateRoomName();
+
+  // @ts-expect-error
+  // TS2445: Property 'rooms' is protected and only accessible within class 'Adapter' and its subclasses.
+  const room = await io.sockets.adapter.rooms.get(roomName);
+  if (room === undefined || room.size < MAX_ROOM_SIZE) {
+    await socket.join(roomName);
+
+    // TODO: Use a common logging format
+    console.info(`[CREATE] Client created and joined room ${roomName}`);
+    socket.emit('create-room-success', roomName);
+    return true;
```

Comment:
do we need to return true / false in those function? what's the logic? Are those returns used anywhere?

Reply:
We don't need to, but it's no harm either. They could potentially be used in the future. Happy to remove now.

Reply:
yeah I think it's better because returning true / false it was a bit confusing, returning in general it's cool to avoid the extra if, but if not needed maybe better not to return at all